pallets/astar-xcm-benchmarks/src/mock.rs

```diff
@@ -119,43 +116,31 @@ impl frame_system::Config for Test {
     type Lookup = IdentityLookup<Self::AccountId>;
     type RuntimeEvent = RuntimeEvent;
     type BlockHashCount = BlockHashCount;
-    type Version = ();
     type PalletInfo = PalletInfo;
     type AccountData = pallet_balances::AccountData<u64>;
-    type OnNewAccount = ();
-    type OnKilledAccount = ();
-    type SystemWeightInfo = ();
-    type SS58Prefix = ();
-    type OnSetCode = ();
     type MaxConsumers = ConstU32<16>;
     type RuntimeTask = RuntimeTask;
-    type SingleBlockMigrations = ();
-    type MultiBlockMigrator = ();
-    type PreInherents = ();
-    type PostInherents = ();
-    type PostTransactions = ();
 }
 
 parameter_types! {
     pub const ExistentialDeposit: u64 = 10;
 }
 
+#[derive_impl(pallet_balances::config_preludes::TestDefaultConfig)]
 impl pallet_balances::Config for Test {
```

Comment:
I'm not sure about the impact on actual tests, but perhaps using all default values might work. For example, having `MaxLocks` set to `ConstU32<100>` or **1** as the existential deposit might also work. You can find default values for pallet_balances [here](https://github.com/paritytech/polkadot-sdk/blob/39e71bd58f282508db334a424ca6c2997d94acf5/substrate/frame/balances/src/lib.rs#L216).